src/applications/qlippy-backend/src/settings/clipboard.setting.ts

```
@@ -23,6 +27,99 @@ export const clipboardSettings = createSettings<ClipboardSettings>({
+        }
+    ]),
+    preSaveFn: async (settings) => {
+        const {version, history} = settings;
+
+        if (version < 3) return settings;
+
+        if (isVerbose()) console.time('Clipboard pre save');
+
+        const historyItems = await Promise.all(history.map(async (item) => {
+            if (item.type === 'text') {
+                delete item.text;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)